src/main/java/edu/kis/powp/jobs2d/drivers/gui/DriverManagerDriverChangeObserver.java

```diff
@@ -0,0 +1,15 @@
+package edu.kis.powp.jobs2d.drivers.gui;
+
+import edu.kis.powp.jobs2d.features.DriverFeature;
+import edu.kis.powp.observer.Subscriber;
+
+public class DriverManagerDriverChangeObserver implements Subscriber {
```

Comment:
The name of the class is incorrect. It should explain why is the Observer monitoring a DriverChange in DriverManager. Look this way: all driver manager observers are DriverManagerDriverChangeObserver's. That's why the name must focus on update() goal.